.github/workflows/integration_test.yml

```diff
@@ -0,0 +1,39 @@
+# TODO: Support running all integration tests when it's possible to 
+# run self-hosted runners in parallel.
+name: Integration Test
+
+on:
+  pull_request:
+
+jobs:
+  changed_packages:
+    runs-on: [self-hosted, linux]
+    if: ${{ github.repository_owner == 'flutter-tizen' }}
```

Comment:
Is there any reason to check `repository_onwer` ?

Reply:
I added it to block forked repository from triggering it.